omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -58,6 +60,8 @@
 	AttachmentsContext attachmentsContext = Context.getRegisteredComponent(AttachmentsConstants.COMPONENT_ATT_CONTEXT,
 	    AttachmentsContext.class);
 	
+	ComplexDataHelper complexDataHelper = new ComplexDataHelper1_10();
```

Comment:
You can just do:
```java
attachmentsContext.getComplexDataHelper()
```
You need to understand that this is wrong, it will not resist beyond 2.x. Basically a Spring bean should never be instantiated directly, **never**.

Whilst you're at it make `obsSaver` and `attachmentsContext` `private`.